fairwaves_xtrx.py

```diff
@@ -155,7 +155,7 @@ def __init__(self, sys_clk_freq=int(125e6), with_cpu=True, cpu_firmware=None, wi
             cd         = "pcie"
         )
         self.add_pcie(phy=self.pcie_phy, address_width=64, ndmas=1,
-            with_dma_buffering = True, dma_buffering_depth=16384,
+            with_dma_buffering = True, dma_buffering_depth=16384 if nonpro else 8192,
```

Comment:
I think this is backward; `nonpro` should be 8K.

Reply:
Thanks!